### PR TITLE
mantis privacy support, removed unsupported media types

### PR DESCRIPTION
**Pull request by parisholley**

### CODE REVIEW

File: modules/mantisBidAdapter.js [hunk 1]
```diff
@@ -240,6 +241,11 @@ const spec = {
       }),
       property: property
     };
+
+    if (bidderRequest && bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies) {
+      query.consent = false;
```
Comment (aleksatr) [inline modules/mantisBidAdapter.js:246]: @parisholley what is the intent here, you don't want to collect consent string in GDPR zone (you're not part of the global vendor list)? You just mark this request as if consent was not given?

**Comment by aleksatr** [comment]
hey @parisholley , for any adapter (new or updated) submission, code under review must have at least 80% unit test coverage. From what I see mantisBidAdapter has 54% coverage. Please increase unit test coverage above the 80% threshold. To generate and view unit test coverage:
```
gulp test-coverage
gulp view-coverage
```

File: modules/mantisBidAdapter.js (hunk 1 above)
Comment (parisholley) [reply modules/mantisBidAdapter.js:246]: correct, we don't have plans to track user data in the EU (most of our business focuses on the US), so just prefer to act as if we never have consent (which we do today on any inbound EU requests outside prebid).

**Comment by parisholley** [comment]
@aleksatr how do you recommend we test non-prebid related code? unlike most of the adapters, we have complex viewability checks built into the bidder, which would require the tests to actually execute a stubbed prebid request, write a creative into an iframe, etc... i don't see any examples of how to do this.. there isn't much value in unit testing this code as it is very browser dependent and to truly test, would require cross-browser execution vs only running in headless chrome

**Comment by parisholley** [comment]
@aleksatr done :)

**Comment by aleksatr** [comment]
LGTM

**Pull request merged**